src/dirichlet_multi.jl

```diff
@@ -38,3 +37,6 @@ function posterior_canon(pri::Dirichlet, G::Type{Multinomial}, x::Matrix{T}, w::
 	DirichletCanon(a)
 end
 
+# missing method
+posterior(pri::Dirichlet{T}, G::IncompleteFormulation, x) where {T} = Base.convert(Dirichlet{T}, posterior_canon(pri, G, x))
+posterior(pri::Dirichlet{T}, G::IncompleteFormulation, x, w) where {T} = Base.convert(Dirichlet{T}, posterior_canon(pri, G, x, w))
```

Comment:
It seems the static type parameter could be removed?
```suggestion
posterior(pri::Dirichlet{T}, G::IncompleteFormulation, x) where {T} = Base.convert(Dirichlet{T}, posterior_canon(pri, G, x))
posterior(pri::Dirichlet{T}, G::IncompleteFormulation, x, w) where {T} = Base.convert(Dirichlet{T}, posterior_canon(pri, G, x, w))
```
```suggestion
posterior(pri::Dirichlet, G::IncompleteFormulation, x) = convert(typeof(pri), posterior_canon(pri, G, x))
posterior(pri::Dirichlet, G::IncompleteFormulation, x, w) = convert(typeof(pri), posterior_canon(pri, G, x, w))
```
Generally, however, I wonder if these methods point to a bug in `posterior_canon` and one should just make sure that they return the correct type (similar to how we started to fix `fit` in Distributions and make it respect the specified type parameters).